packages/flutter_tools/lib/src/flutter_device_manager.dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'package:meta/meta.dart';
+import 'package:process/process.dart';
+
+import 'android/android_device_discovery.dart';
+import 'android/android_sdk.dart';
+import 'android/android_workflow.dart';
+import 'artifacts.dart';
+import 'base/config.dart';
+import 'base/file_system.dart';
+import 'base/logger.dart';
+import 'base/os.dart';
+import 'base/platform.dart';
+import 'base/terminal.dart';
+import 'base/user_messages.dart' hide userMessages;
+import 'device.dart';
+import 'features.dart';
+import 'fuchsia/fuchsia_device.dart';
+import 'fuchsia/fuchsia_sdk.dart';
+import 'fuchsia/fuchsia_workflow.dart';
+import 'ios/devices.dart';
+import 'ios/ios_workflow.dart';
+import 'ios/simulators.dart';
+import 'linux/linux_device.dart';
+import 'macos/macos_device.dart';
+import 'macos/macos_workflow.dart';
+import 'macos/xcode.dart';
+import 'tester/flutter_tester.dart';
+import 'version.dart';
+import 'web/web_device.dart';
+import 'windows/windows_device.dart';
+import 'windows/windows_workflow.dart';
+
+
+class FlutterDeviceManager extends DeviceManager {
```

Comment:
The only purpose of this class is to construct dependencies for the injection, it can be one of the last things we migrate